Source/ASDisplayNode+Layout.mm

```diff
@@ -75,19 +76,19 @@ - (ASLayout *)layoutThatFits:(ASSizeRange)constrainedSize parentSize:(CGSize)par
     ASDisplayNodeAssertNotNil(_calculatedDisplayNodeLayout->layout, @"-[ASDisplayNode layoutThatFits:parentSize:] _calculatedDisplayNodeLayout->layout should not be nil! %@", self);
     // Our calculated layout is suitable for this constrainedSize, so keep using it and
     // invalidate any pending layout that has been generated in the past.
-    _pendingDisplayNodeLayout = nullptr;
+    ASLayoutElementGetCurrentContext().pendingDisplayNodeLayout = nullptr;
```

Comment:
Thanks for pulling this together!

Once we have the tests passing for this change, let's consider creating a #define macro that shortens this expression - it's used in a number of places with fairly long lines, so it would be worth it.